tests/e2e.rs

```diff
@@ -276,7 +276,7 @@ fn test_digest(#[case] uri: &str, #[case] expected_sha: &str) {
     is_empty()
 )]
 #[case::sha_prefix("0169", true, is_empty())]
-#[case::non_existing("non-existing", false, contains("Cannot find policy"))]
+#[case::non_existing("non-existing", false, contains("No such file or directory"))]
```

Comment:
I think we should keep the error message and handle better the error in the code. Because the old error message makes more sense and changing it is in a breaking change.